glossary.md

```diff
@@ -16,6 +16,15 @@ For example, namespaces, resource limits, and mounts are all part of the contain
 ## Container namespace
 
 On Linux, a leaf in the [namespace][namespaces.7] hierarchy in which the [configured process](config.md#process-configuration) executes.
+For example, with a PID namespace hierarchy like:
```

Comment:
This is confusing... Why don't we instead say "If the container init process pid is `100` in the runtime pid namespace, the container init process pid will be `0` inside the containers namespace"...